### PR TITLE
[DOC][MAINT] Standardize docstrings of `plotting` module

### PR DESCRIPTION
**Pull request by NicolasGensollen**
This PR proposes to extend recent work from #2875 to the `plotting` module where a lot of arguments are common to multiple functions. It also improves already standardized docstrings (add object references for example).

### CODE REVIEW

**Review by bthirion — APPROVED** [review]
LGTM, thx !

File: nilearn/plotting/img_plotting.py [hunk 1]
```diff
@@ -927,6 +693,7 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
     return display
 
 
+@fill_doc
 def plot_prob_atlas(maps_img, bg_img=MNI152TEMPLATE, view_type='auto',
```
Comment (tsalo) [inline nilearn/plotting/img_plotting.py:697]: Something in this docstring is causing the following error in the build_docs CI:

>sphinx.errors.SphinxWarning: /home/circleci/project/nilearn/plotting/img_plotting.py:docstring of nilearn.plotting.img_plotting.plot_prob_atlas:52:Inline interpreted text or phrase reference start-string without end-string.

File: nilearn/_utils/docs.py [hunk 2]
```diff
@@ -15,45 +15,50 @@
...
     Override download URL. Used for test only (or if you
     setup a mirror of the data).
     Default=None."""
 
 # Smoothing_fwhm
 docdict['smoothing_fwhm'] = """
-smoothing_fwhm : float, optional.
-    If smoothing_fwhm is not None, it gives the size in millimeters of the
+smoothing_fwhm : :obj:`float`, optional.
+    If `smoothing_fwhm` is not None, it gives the size in millimeters of the
```
Comment (tsalo) [inline nilearn/_utils/docs.py:54]: I think single backticks render oddly with Sphinx, so I generally use double backticks, which render as code snippets.

File: nilearn/plotting/img_plotting.py (hunk 1 above)
Comment (NicolasGensollen) [reply nilearn/plotting/img_plotting.py:697]: Thanks @tsalo ! This should be fixed now.

File: nilearn/_utils/docs.py (hunk 2 above)
Comment (NicolasGensollen) [reply nilearn/_utils/docs.py]: Thanks for the review! I changed the single backticks into double in the standard entries.

**Comment by NicolasGensollen** [comment]
Merging, thanks!

**Pull request merged**